requirements.in

```diff
@@ -2,3 +2,4 @@ langchain
 openai
 click
 python-dotenv
+tiktoken
```

Comment:
what is the need for it?

Reply:
It estimates the cost, basically it calculates the number of token according the model. It is a langchain's optional package